.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Added this so that its possible to run the jest tests in the test explorer in vscode